doc/news/changes/minor/20190808RezaRastak

```diff
@@ -0,0 +1,5 @@
+Improved: CellDataStorage stores a reference to a single unique triangulation
+to ensure that cells from other triangulations cannot store any data on the
+current object.
+<br>
+(Reza Rastak, 2019/08/08)
```

Comment:
This is technically an incompatibility. Could you move the changelog entry to that folder?